src/update_agent/mod.rs

```diff
@@ -238,6 +263,78 @@ impl UpdateAgent {
     }
 }
 
+/// Attempt to broadcast msg to all sessions registered in systemd's login manager.
+/// Returns a Result with a tuple of total sessions found and sessions broadcasted to,
+/// if no error.
+fn broadcast(msg: &str) -> Fallible<(u8, u8)> {
+    let sessions = get_user_sessions()?;
+    let sessions_total = sessions.len() as u8;
+    let mut sessions_broadcasted: u8 = 0;
+
+    let broadcast_msg = format!(
+        "\nBroadcast message from Zincati at {}:\n{}\n",
+        chrono::Utc::now().format("%a %Y-%m-%d %H:%M:%S %Z"),
+        msg
+    );
+
+    // Iterate over sessions and attempt to write to each session's tty.
+    for session in sessions.into_iter() {
+        let user = session.user;
+        let tty_dev = match session.tty {
+            Some(mut tty) => {
+                tty.insert_str(0, "/dev/");
+                tty
+            }
+            None => {
+                log::info!(
```

Comment:
I think this can be demoted to `debug!()`. It seems to be a quite common scenario related to non-interactive users, and there is no need to spam the logs about that.

Reply:
Thanks! I wasn't too familiar with _logged in_ non-interactive users, so I thought they were quite rare.